subsys/bluetooth/host/hci_core.c

```diff
@@ -61,6 +61,9 @@
 
 #define HCI_CMD_TIMEOUT      K_SECONDS(10)
 
+#define PHY_REQ_MAX_RETRY    (5)
+#define PHY_REQ_RETRY_CI     (6)
```

Comment:
No need for the () since what the macros expand to are "atomic" in that there's no way the parenthesis can help ensure the correct evaluation order.

Reply:
That said, are these fixed values really universal, and no one would want them any different? If not, then Kconfig may be the better way.

Reply:
Good question. I just wanted - as a starting point - to avoid polluting the config-space. I think the PHY_REQ_MAX_RETRY could easily be a CONFIG, maybe keeping PHY_REQ_RETRY_CI fixed, as it relates to LLCP instant.